functions/npm.fish

```diff
@@ -1,3 +1,3 @@
-function npm -d "javascript package manager"
+function npm -d "javascript package manager" -c npm -w npm
```

Comment:
I was reading about completions and I noticed that on the official [documentation](http://fishshell.com/docs/current/commands.html#complete) we have this:

> To implement an alias, use the -w or --wraps option:
> 
> `complete -c hub -w git`
> Now hub inherits all of the completions from git. Note this can also be specified in a function declaration.

So we are basically creating an alias name `npm` for `npm`